gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+                  <NxTable.Cell>
+                    Navigation links have icons to their left, this specifies the icon to be used.
+                  </NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>text</NxTable.Cell>
+                  <NxTable.Cell>string</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>The text that should appear in the navigation link</NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>href</NxTable.Cell>
+                  <NxTable.Cell>string</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>URL</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Body>
+            </NxTable>
+          </section>
+          <section className="nx-tile-subsection">
+            <header className="nx-tile-subsection__header">
+              <h3 className="nx-h3"><NxCode>NxGlobalSidebarNavigationLink</NxCode> Props</h3>
+            </header>
+            <p className="nx-p">The following CSS classes are available for use on child elements.</p>
+            <NxTable>
+              <NxTable.Head>
+                <NxTable.Row>
+                  <NxTable.Cell>Name</NxTable.Cell>
+                  <NxTable.Cell>Location</NxTable.Cell>
+                  <NxTable.Cell>Description</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Head>
+              <NxTable.Body>
+                <NxTable.Row>
+                  <NxTable.Cell><NxCode>nx-global-sidebar__expanded-content</NxCode></NxTable.Cell>
+                  <NxTable.Cell>element</NxTable.Cell>
+                  <NxTable.Cell>
+                    Content wrapped with <NxCode>nx-global-sidebar__expanded-content</NxCode> will appear in the
+                    expanded view, but will not be displayed in the collapsed view.
+                  </NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Body>
+            </NxTable>
+          </section>
+        </div>
+      </section>
+      <section className="nx-tile">
+        <header className="nx-tile-header">
+          <div className="nx-tile-header__title">
+            <h2 className="nx-h2">Basic Global Sidebar Example with Nav</h2>
+          </div>
+        </header>
+        <div className="nx-tile-content">
+          <NxP>
+            <a className="nx-text-link" href="#/NxGlobalSidebarExample">
+              Click here to navigate to the live example.
+            </a>
+          </NxP>
+          <CodeExample content={NxGlobalSidebarExample} />
+        </div>
+      </section>
+      <section className="nx-tile">
+        <header className="nx-tile-header">
+          <div className="nx-tile-header__title">
+            <h2 className="nx-h2">Global Sidebar on a page with another page level sidebar</h2>
```

Comment:
I'm not sure this example is necessary here.  I have examples of this in RSC-487 where it was decided whether the old sidebar is supposed to appear next to or underneath of the new header.

Reply:
> I'm not sure this example is necessary here. I have examples of this in RSC-487 where it was decided whether the old sidebar is supposed to appear next to or underneath of the new header.

Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/58ebce60573ddfb22d395e8a8ebca0557f73596d